content/en/docs/Badges/MCH2022/software-development/api/pax_graphics/_index.md

```diff
@@ -174,10 +174,10 @@ drawing. Consider the following example:
 //...
     // Save this for later.
     pax_push_2d(&buf);
-    // Modify the translation: shear it.
-    pax_apply_2d(&buf, matrix_2d_shear(0.5, 0));
-    // This will no longer have a circular shape.
-    pax_draw_circle(&buf, 0xff00ff00, 60, 60, 20);
+        // Modify the translation: shear it.
```

Comment:
It looks like you added the indentation I remove, because I though it was spurious intentionally. I assume it's meant to show which drawing primitives are affected by the transformation matrix.

I think this may be a bit sublte for beginners to grasp, and cause some confusion. I'd suggest adding a block { } around the transformed code and a comment as to why it's intended. (or leave out the indentation :)

Just a suggestion, though, not passionate about it.